Packs/GooglePubSub/Integrations/GooglePubSub/GooglePubSub.py

```diff
@@ -698,6 +698,7 @@ def publish_message_command(
     project_id: str,
     data: str = None,
     attributes: str = None,
+    delim_char_attributes: str = None,
```

Comment:
Let's change this to be default ',' if possible. Shouldn't make a difference as the default in yml is ',' but in case that is not the case this None could ovveride the next default in attribute_pairs_to_dict

Reply:
Done!